install/kubernetes/helm/istio/charts/tracing/templates/gateway.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-tracing-gateway
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    istio: {{ .Values.gatewayName }}
```

Comment:
What is the use of this `selector`?

Reply:
Just copied this gateway from telemetry-gateway, which uses[ this approach](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/telemetry-gateway/templates/gateway.yaml#L9).

Reply:
ok, thanks @objectiser . I saw @douglas-reid add this, so want to get an answer from @douglas-reid for why we need this `selector`?

Reply:
I answered this in a different PR, but that selector was requested by reviewers when I add the telemetry-gateway, either directly in the PR or over Slack or similar. The intent, iirc, was to allow exposing telemetry via an ilb vs. ingressgateway or similar.

Reply:
seems like overkill, but can be cleaned up in a separate PR.  I'd like to remove these selectors (to remove config variables) in all of the telemetry gateways.